OfekQuery.html

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/bootstrap.css">
+	<link rel="stylesheet" type="text/css" href="css/index.css">
+	<link rel="stylesheet" type="text/css" href="css/OfekQuery.css">
+
+</head>
+
+<body>
+
+<!-- Nav Bar -->
+<nav class="navbar navbar-default nav-color">
+	<div class="container-fluid">
```

Comment:
Usually, Bootstrap containers should not be nested.
http://getbootstrap.com/docs/4.0/layout/overview/